nilearn/datasets/atlas.py

```diff
@@ -576,3 +576,73 @@ def fetch_yeo_2011_atlas(data_dir=None, url=None, resume=True, verbose=1):
         url=url,
         resume=resume,
         verbose=verbose)
+
+
+def fetch_atlas_aal(data_dir=None, url=None, resume=True, verbose=1):
+    """Download and return file names for the AAL for SPM 12 parcellation.
+
+    The provided images are based on the spatially normalized single-subject
```

Comment:
From that sentence I have the impression that AAL was computed on the MNI T1 template. You can simply say `This atlas has been registered to MNI space` or something like that. Don't we say that for other atlases? If no, then we should.